mpf/platforms/system11.py

```diff
@@ -310,9 +332,6 @@ def _enable_a_side(self):
                 if self.c_side_active:
                     self._disable_all_c_side_drivers()
                     self._disable_ac_relay()
-                    self.delay.add(ms=self.system11_config['ac_relay_delay_ms'],
```

Comment:
This delay call seems redundant to me, because `self._disable_ac_relay()` also creates a delay for `ac_relay_delay_ms` that uses `self._a_side_enabled` as a callback.

This block here uses the same delay to call `self._enable_a_side`, which would also call `self._a_side_enabled`. I've noticed no issues removing the redundant call, but please let me know if I'm overlooking something.